concourse/client/api.py

```diff
@@ -160,6 +160,14 @@ def unpause_pipeline(self, pipeline_name: str):
                 body=""
         )
 
+    @ensure_annotations
+    def pause_pipeline(self, pipeline_name: str):
+        pause_url = self.routes.pause_pipeline(pipeline_name)
+        self.request_builder.put(
+                pause_url,
+                body=""
```

Comment:
nit: can this not be omitted (empty body)?

Reply:
also: sure it's `PUT` rather than `POST`?

Reply:
yes PUT I checked go routes